src/backend/distributed/commands/extension.c

```diff
@@ -491,6 +494,115 @@ PreprocessAlterExtensionUpdateStmt(Node *node, const char *queryString)
 }
 
 
+/*
+ * PostprocessAlterExtensionUpdateStmt is called after ALTER EXTENSION UPDATE
+ * command is executed by standard utility hook.
+ *
+ * Actually, we do not need such a post process function for ALTER EXTENSION
+ * UPDATE commands unless the extension is Citus itself. This is because we
+ * need to mark existing objects that are not included in distributed object
+ * infrastructure in older versions of Citus (but now should be) as distributed
+ * if we can update Citus successfully via standard utility hook.
+ */
+List *
+PostprocessAlterExtensionUpdateStmt(Node *node, const char *queryString)
+{
+	if (IsCreateAlterExtensionUpdateCitusStmt(node))
```

Comment:
This also includes CREATE EXTENSION, but we only care about ALTER EXTENSION. In case of CREATE EXTENSION, pg_dist_object will always be empty.